src/main/resources/static/javascript/mainApp/app.service.js

```diff
@@ -89,7 +89,7 @@
                     .then((response) => {
                         callback(response.data);
                     }, (response) => {
-                        callError(response);
+                        Error(response);
```

Comment:
I believe this should stay as callError.

Reply:
Changed it back to callError. Also, there were a handful of codacy issues complaining about the amount of indentations, but they are gone now and I don't think I fixed them.